project_ux/README.rst

```diff
@@ -0,0 +1,65 @@
+.. |company| replace:: ADHOC SA
+
+.. |company_logo| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-logo.png
+   :alt: ADHOC SA
+   :target: https://www.adhoc.com.ar
+
+.. |icon| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-icon.png
+
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==========
+Project UX
+==========
+
+
+Several improvements to project:
+
+#. Make Dates and Analytic Account fields visible on project form view.
+#. Add template in tasks for auto complete the data.
+
+Installation
```

Comment:
This sections are empty, will be good if they have content as it is in the README guideline template.
Would be good if could look like the next one:

![image](https://user-images.githubusercontent.com/7593953/41478402-eb611cda-709d-11e8-8c6e-907118962a54.png)